client/landing/stepper/declarative-flow/internals/steps-repository/playground/lib/resolve-blueprint-from-url.ts

```diff
@@ -105,9 +105,11 @@ export async function resolveBlueprintFromURL( url: URL ) {
 	blueprint.preferredVersions!.wp =
 		query.get( 'wp' ) || blueprint.preferredVersions!.wp || 'latest';
 
-	// Features
+	// Features, must have networking enabled.
```

Comment:
@zaerl I haven't reviewed & understood this change yet, but we should also copy this change over to relevant PG file, where this entire file is copied from and I think we should strive for them to not be different.

Reply:
It makes sense. I think networking should be enabled by default, so that a playground is more similar to a WoA site where you can install plugins.

Reply:
I handled this in https://github.com/Automattic/wp-calypso/pull/103166